tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -678,9 +677,7 @@ describe('WidgetBuilder', function () {
       widgets: [widget],
     };
 
-    const handleSave = jest.fn();
```

Comment:
was this not being used?

Reply:
I split it up into two tests, this one just ensures the setup is correct, the one below checks the mock is called properly https://github.com/getsentry/sentry/pull/33453/files#diff-d93e71c8592407cd2356e2af95672ea0ed63254553dc2f3eeb591f21c42bfdb6R762

Reply:
So yeah, it's not being used in this test